src/blog/2023/07/flowforge-1-9-release.md

```diff
@@ -0,0 +1,81 @@
+---
+title: FlowForge now offers API Documentation with Swagger UI
+subtitle: FlowForge 1.9 adds new features to make it easier to administer FlowForge
+description: The new release of FlowForge 1.9 adds new API Swagger documentation and the ability to customize Node-RED pallettes.
+date: 2023-07-06 
+authors: ["ian-skerrett"]
+image: 
+tags:
+    - posts
+    - flowforge
+    - releases
+---
+
+FlowForge 1.9 adds new features to make it easier to adminster FlowForge platform deployments, including new API documentation and the ability to create customized Node-RED palettes.
+
+<!--more-->
+
+## API Documentation
+FlowForge API allows developers to progammatically interact with the FlowForge platform. This makes it possible to integrate FlowForge into different infrastrcutre technologies, create scripts to automate specifc FlowForge tasks and embedd FlowForge into other applications.  
+
+In the 1.9 release we are now publishing our [API documentation](https://flowforge.com/docs/api/) using the [OpenAPI specification](https://swagger.io/specification/) and making it viewable with the Swagger UI. Both these industrial standards will make using the FlowForge API easier to use and understand.
+
+
+## Customize Node-RED Palettes  [#2002](https://github.com/flowforge/flowforge/issues/2002)
+
+FlowForge platform adminstrators are now able to create customized Node-RED palettes that will be used when a Node-RED instance is created. An adminstator can create pre-defined templates to specify the nodes that should be included in the palette. This makes it easier for FlowForge teams to standardized on Node-RED usage across an organization.
+
+Note: this feature is not available for FlowForge cloud users since they do not have adminstrator access. In the future, the ability to create templates will be enabled for teams and this will allow for customize Node-RED palettes for individual teams.
+
+Starting with the 1.9 release, FlowForge Cloud will begin to include the [Node-RED debugger](https://flows.nodered.org/node/node-red-debugger) and [lint node](https://github.com/node-red/nrlint) in their default templates. 
```

Comment:
```suggestion
Starting with the 1.9 release, FlowForge Cloud will begin to include the [Node-RED debugger](https://flows.nodered.org/node/node-red-debugger) and [Node-RED Flow Linter](https://github.com/node-red/nrlint) in their default templates. 
```

Reply:
How does this work with pipelines? Will these be copied to snapshots and to production instances? Debugger not linter should be there by default?

@MarianRaphael will this break deploys for customers that use a module cache?

Reply:
```suggestion
```

Reply:
Clarified via Slack. Will not going to happen in 1.9